<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AOD_GUI</class>
 <widget class="QDialog" name="AOD_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Siglent_AWG</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>关于x的表达式:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="expr_plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>551</width>
     <height>131</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>sin(2*pi*x)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始计算波形</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>571</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="mplwidget" name="mplwidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>551</width>
     <height>181</height>
    </rect>
   </property>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>170</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="save_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存.bin文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="read_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>读取.bin文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload_pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>620</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上传波形</string>
   </property>
  </widget>
  <widget class="QLabel" name="save_filename">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>480</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存文件名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="read_filename">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>读取文件名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="update_state">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>620</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上传状态:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>重复频率/kHz:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>波形长度/Sa:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_repeat">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="data_len">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>1200</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>波形幅度/Vpp:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="data_amp">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CH_comboBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>580</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>CH1</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <item>
    <property name="text">
     <string>CH1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CH2</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="dev_comboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>580</y>
     <width>331</width>
     <height>28</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="dev_refresh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新设备</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="data_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>620</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>temp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>波形名称:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CH1:</string>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>650</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="CH1_out_pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>670</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开启</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CH1_out_pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>670</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CH2_out_pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>670</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开启</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CH2_out_pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>670</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>670</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CH2:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="bin_checkBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>620</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>不使用bin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CH_out_pushButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>670</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>均关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CH_out_pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>670</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>均开启</string>
   </property>
  </widget>
  <widget class="QPushButton" name="freq_up_pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>更新并上传重频</string>
   </property>
  </widget>
  <widget class="QPushButton" name="amp_up_pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>更新并上传幅度</string>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mplwidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
